tests/GridSelect.hs

```diff
@@ -0,0 +1,12 @@
+module GridSelect where
+
+import Test.Hspec
+
+import XMonad.Actions.GridSelect
+
+spec :: Spec
+spec = do
+  describe "stringToRatio" $ do
+    it "returns values in (0, 1) range" $ do
+      stringToRatio "deadbeef" `shouldSatisfy` (>= 0)
+      stringToRatio "deadbeef" `shouldSatisfy` (<= 1)
```

Comment:
Since there is something like `arbitraryPrintableChar` in QuickCheck it might be nice to property test this instead?

Reply:
I used default String instance, not `arbitraryPrintableChar` - is that okay?

Reply:
That's totally okay as well